src/components/standalone/NostromoCrew.vue

```diff
@@ -44,14 +46,20 @@
         </template>
         <template #item.name="{item}">
           <div class="font-size-15 py-2 text-grey-darken-2">
-            <span v-if="item.firstName || item.lastName">
-              {{ item.lastName }}<span v-if="item.firstName && item.lastName">, </span>{{ item.firstName }}
+            <img class="profile-image" :src="item.image" :alt="item.firstName">
+            <span v-if="item.firstName || item.lastName" class="profile-name">
+              <OutboundLink :href="`https://www.berkeley.edu/directory/?search-term=${item.firstName}+${item.lastName}`">
+                <span v-if="item.firstName && item.lastName">
+                  {{ item.firstName }} {{ item.lastName }}
+                </span>
```

Comment:
@mohamalnadi this is an edgy case, but it seems like if we have `firstName` or `lastName` but not both, then `item.firstName || item.lastName` would evaluate true but `item.firstName && item.lastName` would evaluate false, meaning nothing at all would render in the name column - not a name and not a dash.

Would it maybe make sense to take out the `item.firstName && item.lastName` conditional, and instead have fallbacks at the point where the fields are interpolated? If we did the interpolation as `{{ item.firstName || '' }}`, for example, that avoids the scenario where a null value turns into the string "null."